sources/packages/web/src/views/student/financial-aid-application/FinancialInfo.vue

```diff
@@ -93,13 +93,17 @@
           </ContentGroup>
         </Section>
       </ContentGroup>
-      <FooterNavigator previous="select-program" next="confirm-submission" />
+      <FooterNavigator
+        v-bind:previous="selectProgram"
```

Comment:
Please consider using the shortcut for v-bind (:previous) instead (v-bind:previous).

Reply:
I just like using v-bind because it's more explicit. A little old school in this. Hope this is ok